sample/src/main/res/layout/activity_main.xml

```diff
@@ -270,6 +270,15 @@
             android:id="@+id/transactionLimitMaxET"
             android:hint="Transaction limit Max" />
 
+        <androidx.appcompat.widget.AppCompatEditText
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="10dp"
+            android:inputType="numberDecimal"
+            android:id="@+id/feesView"
+            android:text="1.5"
```

Comment:
Let's not have default value in here not to confuse QA in case they don't explicitly provide anything.